lib/beautify-stack.js

```diff
@@ -1,13 +1,19 @@
 'use strict';
 
-function beautifyStack(stack) {
-	var re = /(?:^(?! {4}at\b).{6})|(?:\((?:[A-Z]:)?(?:[\\\/](?:(?!node_modules[\\\/]ava[\\\/])[^:\\\/])+)+:\d+:\d+\))/;
-	var found = false;
+var StackUtils = require('stack-utils');
+
+var stackUtils = new StackUtils({
+	internals: StackUtils.nodeInternals().concat([
+		/\(internal\/[\w_-]+\.js:[0-9]+:[0-9]+\)$/,
+		/\(native\)$/,
+		/[\\\/]ava[\\\/](?:lib[\\\/])?[\w-]+\.js:[0-9]+:[0-9]+\)?$/,
+		/[\\\/]node_modules[\\\/](?:bluebird|empower-core)[\\\/]/
```

Comment:
Can you normalize the paths in stack traces to forward slash? That way they're consistent on all systems and with the added benefit of simpler regex here.